Base/Dockerfile

```diff
@@ -70,12 +70,12 @@ COPY supervisord.conf /etc
 #==========
 RUN  mkdir -p /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
   && touch /opt/selenium/config.toml \
-  && chmod -R 777 /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor /etc/passwd \
+  && chmod -R 775 /opt/selenium /opt/selenium/assets /var/run/supervisor /var/log/supervisor /etc/passwd \
   && wget --no-verbose https://github.com/SeleniumHQ/selenium/releases/download/selenium-4.7.0/selenium-server-4.7.2.jar \
     -O /opt/selenium/selenium-server.jar \
   && wget --no-verbose https://repo1.maven.org/maven2/org/seleniumhq/selenium/selenium-http-jdk-client/4.7.2/selenium-http-jdk-client-4.7.2.jar \
     -O /opt/selenium/selenium-http-jdk-client.jar \
-  && chgrp -R 0 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
+  && chgrp -R 1201 /opt/selenium ${HOME} /opt/selenium/assets /var/run/supervisor /var/log/supervisor \
```

Comment:
I believe this would break when using the images in OpenShift, https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html#openshift-specific-guidelines